connector/src/main/java/org/jboss/as/connector/subsystems/datasources/DataSourceModelNodeUtil.java

```diff
@@ -282,13 +282,14 @@ static ModifiableXaDataSource xaFrom(final OperationContext operationContext, fi
         final String username = ModelNodeUtil.getResolvedStringIfSetOrGetDefault(operationContext, dataSourceNode, USERNAME);
         final String password= ModelNodeUtil.getResolvedStringIfSetOrGetDefault(operationContext, dataSourceNode, PASSWORD);
         final String securityDomain = ModelNodeUtil.getResolvedStringIfSetOrGetDefault(operationContext, dataSourceNode, SECURITY_DOMAIN);
-        final boolean elytronEnabled = ModelNodeUtil.getBooleanIfSetOrGetDefault(operationContext, dataSourceNode, ELYTRON_ENABLED);
+        if (securityDomain != null) {
+            ConnectorLogger.DS_DEPLOYER_LOGGER.legacySecurityNotSupported();
```

Comment:
This method produces an OperationFailedException but it is not thrown. Here an OFE is ok.

Reply:
fixed